packages/block-editor/src/components/inserter/style.native.scss

```diff
@@ -51,13 +51,12 @@
 
 .inserter-tabs__wrapper {
 	overflow: hidden;
+	height: 100%;
```

Comment:
I believe this change and usage of `height: 100%` conflicts with our fix for the [scroll view height](https://github.com/WordPress/gutenberg/pull/34129/commits/e41a6929362936ce71898faa86b6a653744e8cb3). We should likely use that approach instead, otherwise we may run into incorrect height issues again. 

I tested applying those change rudimentarily with the following patch, and the results worked as expected. 

<details><summary>Flex Alternative</summary>

```diff
diff --git a/packages/block-editor/src/components/inserter/menu.native.js b/packages/block-editor/src/components/inserter/menu.native.js
index 198bb76132..5f55671445 100644
--- a/packages/block-editor/src/components/inserter/menu.native.js
+++ b/packages/block-editor/src/components/inserter/menu.native.js
@@ -204,7 +204,10 @@ function InserterMenu( {
 		>
 			<BottomSheetConsumer>
 				{ ( { listProps } ) => (
-					<TouchableHighlight accessible={ false }>
+					<TouchableHighlight
+						accessible={ false }
+						style={ { flex: 1 } }
+					>
 						{ ! showTabs || filterValue ? (
 							<InserterSearchResults
 								rootClientId={ rootClientId }
diff --git a/packages/block-editor/src/components/inserter/style.native.scss b/packages/block-editor/src/components/inserter/style.native.scss
index b74e6bf03f..5b30217eb7 100644
--- a/packages/block-editor/src/components/inserter/style.native.scss
+++ b/packages/block-editor/src/components/inserter/style.native.scss
@@ -51,7 +51,7 @@
 
 .inserter-tabs__wrapper {
 	overflow: hidden;
-	height: 100%;
+	flex: 1;
 }
 
 .inserter-tabs__container {
```

</details>

Reply:
> I believe this change and usage of height: 100% conflicts with our fix for the scroll view height. We should likely use that approach instead, otherwise we may run into incorrect height issues again. I.e. we may should just cherry pick the commit to this branch.

Thanks David, I opted to use your example snippet over cherry picking the existing commit. Cherry picking brought in some other changes that I'd prefer stay with #34129

Reply:
👍🏻 We'll just need to resolve the conflict to merge https://github.com/WordPress/gutenberg/pull/34129. Using the class instead of inline styles is likely preferred at that point.